packages/edit-site/src/components/dataviews/add-filter.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	privateApis as componentsPrivateApis,
+	Button,
+	BaseControl,
+	Icon,
+} from '@wordpress/components';
+import { chevronRightSmall, plus } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const {
+	DropdownMenuV2,
+	DropdownSubMenuV2,
+	DropdownSubMenuTriggerV2,
+	DropdownMenuItemV2,
+} = unlock( componentsPrivateApis );
+
+export default function AddFilter( { filters, onChangeView } ) {
```

Comment:
The filters here is "fields" right? Should we stick with its original name?

Reply:
I see from the code that it's actually a custom object constructed from the fields and the filters of the view, I would have just passed "view" and "fields" personally. Maybe at some point we want to show the visible filters as "grayed out" or allow multiple usage of the same field (different operators)...

Reply:
Done as part of https://github.com/WordPress/gutenberg/pull/55992/commits/995ec2b34f4cc53ab766a3d87a88ae8be6ec7ee5